src/GitPrompt.ps1

```diff
@@ -360,9 +487,9 @@ $PoshGitVcsPrompt = {
     catch {
         $s = $Global:GitPromptSettings
         if ($s) {
-            Write-Prompt $s.BeforeText -BackgroundColor $s.BeforeBackgroundColor -ForegroundColor $s.BeforeForegroundColor
-            Write-Prompt "Error: $_" -BackgroundColor $s.ErrorBackgroundColor -ForegroundColor $s.ErrorForegroundColor
-            Write-Prompt $s.AfterText -BackgroundColor $s.AfterBackgroundColor -ForegroundColor $s.AfterForegroundColor
+            Write-Prompt $s.BeforeText
+            Write-Prompt "Error: $_" -BackgroundColor $s.ErrorColor.BackgroundColor -ForegroundColor $s.ErrorColor.ForegroundColor
```

Comment:
Yeah, I can do that.

Reply:
Nice. In updating this, I found a bug in our Write-VcsStatus handling i.e. it would return a collection of string (in ANSI mode) and that trips up the prompt function.